nodeJS/express/introduction_to_express.md

```diff
@@ -0,0 +1,98 @@
+### Introduction
+
+In the previous lesson, you got up and running with Node. You learned how to read and write data from files on the server, and use those to serve up a multi-page website. You may have found this a somewhat verbose and cumbersome process and may have even wondered how on Earth we would manage more complex procedures.
```

Comment:
Do you mean previous lesson or previous section?

Also, not sure about the word "procedures", let's go with "use cases" or something similar: 
```suggestion
In the previous lesson, you got up and running with Node. You learned how to read and write data from files on the server, and use those to serve up a multi-page website. You may have found this a somewhat verbose and cumbersome process and may have even wondered how on Earth we would manage more complex use cases.
```